turbo/snapshotsync/block_snapshots.go

```diff
@@ -1622,32 +1670,47 @@ RETRY:
 				if !bodyGetter.HasNext() {
 					return fmt.Errorf("not enough bodies")
 				}
+
 				bodyBuf, _ = bodyGetter.Next(bodyBuf[:0])
 				if err := rlp.DecodeBytes(bodyBuf, body); err != nil {
 					return err
 				}
+				headerBuf, _ = headerGetter.Next(headerBuf[:0])
+				if err := rlp.DecodeBytes(headerBuf, header); err != nil {
+					return err
+				}
+
 				blockNum++
 			}
-
+			txTypeAndlengthOfAddress := 21
```

Comment:
1 it's not txType - it's first byte for txnHash (to reduce false-positives produced by our indices).